plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/models/IRunningPod.java

```diff
@@ -0,0 +1,16 @@
+/*******************************************************************************
+ * Copyright (c) 2016 Red Hat, Inc. Distributed under license by Red Hat, Inc.
+ * All rights reserved. This program is made available under the terms of the
+ * Eclipse Public License v1.0 which accompanies this distribution, and is
+ * available at http://www.eclipse.org/legal/epl-v10.html
+ * 
+ * Contributors: Red Hat, Inc.
+ ******************************************************************************/
+package org.jboss.tools.openshift.internal.ui.models;
+
+/**
+ * Running pods may be adapted to this interface with Adapters.adapt().
+ */
+public interface IRunningPod {
```

Comment:
I renamed interface to IRunningPodHolder, but now I have an idea that we still can keep it simple. We can have interface IRunningPod empty, without any method, and without an implication if it is a wrapper or a real pod. It can be used now just as a marker that an object is adaptable to a running pod, nothing more. That is mostly needed for Eclipse command framework to decide if the action is to be added to the context menu. But handler does not have to adapt, it can just make simple needed checks, as it is done now. So, the less wrappers/holders, the better, they encumber models.